conf/deployment/vsphere/upi_1az_rhcos_3m_3w_prod_memory.yaml

```diff
@@ -11,3 +11,4 @@ ENV_DATA:
   master_num_cpus: '4'
   master_memory: '16384'
   compute_memory: '65536'
+  fio_storageutilization_min_mbps: 10.0
```

Comment:
This should be part of your fio_workload() config, specific to your test ?

Reply:
I expect that any fio workload jobs which are writing data on ceph based volumes will use this value to compute a timeout of the operation. So far, there is only one such fio job type, which is fio storageutilization set of fixtures. That said, we could rename this option into something like `ceph_write_min_mbps` to encourage others to take this into account as well, but so far the only workloads writing serious amount of data (over 80% of total capacity eg. for alerting tests) were these workload fixtures.

The main reason for having this minimal write speed here is to be able to specify a minimal write speed on each platform/cluster configuration. Infra team could tweak it based on expectations with cluster configuration in mind.

Reply:
The other fio test we have in workload doesn't really check for min bandwidth, I think its specific to individual tests, Also fio dumps the output per job and each job can differ in terms of minimum bandwidth achieved for that io profile. I think the parsing should be left for individual tests for better because control. ex: https://github.com/red-hat-storage/ocs-ci/blob/master/tests/manage/pv_services/test_run_io_multiple_pods.py#L49

Reply:
That is correct, particular performance of writing depends on how exactly it's done. This number is a lower bound for a single thread writer used to determine a timeout, which is especially important when large amount of data are being written (over 50% or even 80% of total OCS capacity).

Let's keep the name as it is then.